test/ruby/test_array.rb

```diff
@@ -114,6 +114,9 @@ def test_misc_0
     assert_equal('1', (x * 1).join(":"))
     assert_equal('', (x * 0).join(":"))
 
+    assert_instance_of(Array, (@cls[] * 5))
```

Comment:
Did you consider adding an `assert_equal_instance` that would call both `assert_equal` and `assert_instance_of`? Could be less verbose.

Reply:
Agreed, I'll make that change.